tests/test_endpoints.py

```diff
@@ -123,7 +125,7 @@ def test_events_endpoint_accepts_valid_mds_payload_pds_events(
     )
     update_response.raise_for_status()
 
-    for pds_mds_event in pds_mds_event_list:
+    for pds_mds_event in mds_event_list:
```

Comment:
you've stripped pds from the source object, should the iterator now be called mds_event not pds_event
```suggestion
    for mds_event in mds_event_list:
```

Reply:
updated in: https://github.com/NHSDigital/multicast-notification-service/pull/137/commits/bf6088526f6106fd3fe85a45c905fc96f9e364a2